srcpkgs/oracle-jdk-11/template

```diff
@@ -0,0 +1,115 @@
+# Template file for 'oracle-jdk-11'
+pkgname=oracle-jdk-11
+version=11.0.2
+_longVersion=11.0.2+9
+revision=1
+_tag=f51449fcd52f4d52b93a989c5c56ed3c
+short_desc="Java Development Kit (JDK)"
+maintainer="Dominic Monroe <monroef4@googlemail.com>"
+homepage="http://www.oracle.com/technetwork/java/javase/downloads/index.html"
+license="oracle"
+lib32disabled=yes
+repository="nonfree"
+only_for_archs="x86_64"
+depends="hicolor-icon-theme desktop-file-utils xdg-utils shared-mime-info wget"
+hostmakedepends="wget"
+provides="java-environment-${version}_1"
+_filename="jdk-11_linux-x64_bin.tar.gz"
+distfiles="http://download.oracle.com/otn-pub/java/jdk/${version}/${_tag}/jdk-${version}_linux-x64_bin.tar.gz"
```

Comment:
I believe correct url is `http://download.oracle.com/otn-pub/java/jdk/${_longVersion}/${_tag}/jdk-${version}_linux-x64_bin.tar.gz`